src/plugins/ui/evaluation_chart.ts

```diff
@@ -232,6 +234,14 @@ export class EvaluationChartPlugin extends UIPlugin {
     }
     for (const [dsIndex, ds] of Object.entries(definition.dataSets)) {
       let label: string;
+      if (
+        this.getters
+          .getHiddenColsGroups(ds.dataRange.sheetId)
+          .flat()
+          .includes(ds.dataRange.zone.left)
```

Comment:
```suggestion
        this.getters.isColHidden(ds.dataRange.zone.left)
```
or does that not exist in 15.0 ?

Reply:
No, it doesn't exist in 15.0 :/